torch_xla/csrc/ops/ops.cpp

```diff
@@ -151,6 +151,18 @@ NodePtr NllLossBackwardOp(const Value& logits, const Value& labels) {
                             std::move(lower_fn));
 }
 
+NodePtr NotSupportedOp(c10::Symbol node_symbol) {
```

Comment:
This op should take a shape.

Reply:
Where would it get the shape from? If there's no XLA lowering, we'd have to write a type equation, which is overkill for something we don't want to implement in the first place.

Reply:
It's the same shape as the output, with int64 type AFAICT.
The int64 is a linearized coordinates of the winning index at that position.